Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3572,14 +3639,42 @@ def panorama_list_rules_command(args: dict):
 
     pretty_rules = prettify_rules(rules, target)
 
+    to_human_readable = {
+        'Location': 'Location',
```

Comment:
you can make the call site  `lambda x: to_human_readable.get(x, x)` and then leave everything out of the dict that doesnt need to be changed

Reply:
example Packs.CrowdStrikeFalconSandbox.Integrations.CrowdStrikeFalconSandboxV2.CrowdStrikeFalconSandboxV2.crowdstrike_analysis_overview_summary_command